deployments/kubernetes/reloader.yaml

```diff
@@ -104,6 +135,14 @@ spec:
             resourceFieldRef:
               divisor: "1"
               resource: limits.memory
+        - name: RELOADER_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        
+        - name: RELOADER_DEPLOYMENT_NAME
+          value: default
+              
```

Comment:
these are generated by pipeline if i am not wrong. pls take the command from pipeline and run it for this to be generated by kustomize